api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -265,7 +264,7 @@ void shouldSendNoLongerEligibleEmailWhenEligibleWithNoChildrenOnFeedAndNotPregna
         PaymentCycle newCycle = repositoryMediator.getCurrentPaymentCycleForClaim(claim);
         assertPaymentCycleWithNoPayment(newCycle, currentPaymentCycleChildrenDobs);
 
-        assertStatusOnClaim(claim, ClaimStatus.PENDING_EXPIRY);
+        assertStatusOnClaim(claim, ClaimStatus.EXPIRED);
```

Comment:
We got the status wrong in the integration test, this scenario should be setting the status to EXPIRED.